<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartMenu</class>
 <widget class="QWidget" name="StartMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Map Size:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select Difficulty:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="difficulty_select">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Easy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Normal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hard</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="start_game">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>440</y>
     <width>571</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Game</string>
   </property>
  </widget>
  <widget class="QComboBox" name="map_size_select">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Small (100*60)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Middle (120*70)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Big (140*80)</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Leaderboard">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>151</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>Leaderboard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>81</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>Logs</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
